sdk/core/core-rest-pipeline/src/policies/retryPolicy.ts

```diff
@@ -113,7 +113,7 @@ export function retryPolicy(
             strategyLogger.info(
               `Retry ${retryCount}: Retry strategy ${strategy.name} retries after ${retryAfterInMs}`
             );
-            await delay(retryAfterInMs);
+            await delay(retryAfterInMs, undefined, { abortSignal: request.abortSignal });
```

Comment:
It always feels cumbersome to have to pass in undefined in the middle.  Can we make `value` (or valueToResolve) part of the options bag?

Reply:
my impression is that only the first parameter is used most of time.  But no strong feeling.  This is not public.